[2025-09-13T10:20:19.707+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-13T10:20:19.777+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.analyze manual__2025-09-13T10:20:05.716383+00:00 [queued]>
[2025-09-13T10:20:19.794+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.analyze manual__2025-09-13T10:20:05.716383+00:00 [queued]>
[2025-09-13T10:20:19.795+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-09-13T10:20:19.819+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): analyze> on 2025-09-13 10:20:05.716383+00:00
[2025-09-13T10:20:19.836+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'analyze', 'manual__2025-09-13T10:20:05.716383+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmph5uslfr5']
[2025-09-13T10:20:19.842+0000] {standard_task_runner.py:105} INFO - Job 234: Subtask analyze
[2025-09-13T10:20:19.841+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=894) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-13T10:20:19.843+0000] {standard_task_runner.py:72} INFO - Started process 895 to run task
[2025-09-13T10:20:20.057+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.analyze manual__2025-09-13T10:20:05.716383+00:00 [running]> on host 6eb00eb09733
[2025-09-13T10:20:20.355+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='analyze' AIRFLOW_CTX_EXECUTION_DATE='2025-09-13T10:20:05.716383+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-13T10:20:05.716383+00:00'
[2025-09-13T10:20:20.356+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-13T10:20:20.379+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-13T10:20:20.382+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/analyze.py']
[2025-09-13T10:20:20.404+0000] {subprocess.py:86} INFO - Output:
[2025-09-13T10:20:22.578+0000] {subprocess.py:93} INFO - /opt/***/scripts/analyze.py:151: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
[2025-09-13T10:20:22.581+0000] {subprocess.py:93} INFO -   today = datetime.utcnow().strftime("%Y-%m-%d")
[2025-09-13T10:20:22.627+0000] {subprocess.py:93} INFO - + Text report saved to /opt/***/data/reports/report_2025-09-13.txt
[2025-09-13T10:20:22.883+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-13T10:20:23.025+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-13T10:20:23.026+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=analyze, run_id=manual__2025-09-13T10:20:05.716383+00:00, execution_date=20250913T102005, start_date=20250913T102019, end_date=20250913T102023
[2025-09-13T10:20:23.116+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-13T10:20:23.209+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-13T10:20:23.214+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
